internal/area/area.go

```diff
@@ -42,16 +42,20 @@ func FromSize(size image.Point) (image.Rectangle, error) {
 // specified percentage of its width. The percentage must be in the range
 // 0 <= heightPerc <= 100.
 // Can return zero size areas.
+//
```

Comment:
Do we want to revert the changes done to the area.go file?

Reply:
Good catch, reverted it.